### PR TITLE
feat: add support to ptvMessage in sendMessage method

### PR DESCRIPTION
**Pull request by arthursimas1**

### CODE REVIEW

**Comment by Salientekill** [comment]
What would be ptvmessage?

**Comment by arthursimas1** [comment]
`ptvMessage` is a video note
image

**Review by azudindaem — APPROVED** [review]
LGTM

**Review by PurpShell — APPROVED** [review]

File: src/Utils/messages.ts [hunk 1]
```diff
@@ -254,6 +255,11 @@ export const prepareWAMessageMedia = async(
 		)
 	})
 
+	if (uploadData.ptv) {
```
Comment (purpshell) [inline src/Utils/messages.ts]:
```suggestion
	if(uploadData.ptv) {
```

Comment (purpshell) [reply src/Utils/messages.ts]: linter 😆

**Review by PurpShell — APPROVED** [review]

**Pull request merged**